ansible_collections/arista/avd/roles/dhcp_provisioner/meta/main.yml

```diff
@@ -8,8 +8,6 @@ galaxy_info:
   galaxy_tags: ['arista', 'network', 'networking', 'eos', 'avd', 'cloudvision', 'cvp']
   platforms:
     - name: EL
-      versions: [7, 8]
+      versions: ["7", "8"]
     - name: Ubuntu
-      versions: [ 20.04 ]
-  dependencies:
-    "arista.cvp": "*"
+      versions: [ "focal" ]
```

Comment:
spaces or no spaces inside `[` and `]` ? ( I vote no spaces as yamllint default is)

Reply:
removed